test/blackbox-tests/test-cases/misc/run.t

```diff
@@ -1,7 +1,6 @@
   $ dune runtest --display short
           diff alias runtest
           diff alias runtest
-          diff alias runtest
```

Comment:
Why the difference here?

Reply:
That's because I moved the existing test from `misc` to `rule-target-inferrence`.